### ISSUE TITLE
pre-model-resnet34 fine-tuning

### ISSUE THREAD

**Issue by dhuqqq** [deepinsight/insightface#162]
Hi
I use the pretrained model(resnet-34),Continue to train on the resnet34 model, why the initial accuracy is still 0
